src/lib.rs

```diff
@@ -597,6 +597,190 @@ impl<A: Array> ArrayVec<A> {
     }
 }
 
+impl<A: Array> ArrayVec<A>
+    where A::Item: Clone,
+{
+    /// Resizes the `ArrayVec` in-place so that `len` is equal to `new_len`.
+    ///
+    /// Does the same thing as `try_resize`, but panics on error instead of
+    /// returning it as a `Result`.
```

Comment:
Documentation for panic should be consistent and visible. Other methods use this style, panic the first word of the line and bold:

```rust
/// ***Panics*** if the `index` is out of bounds.
```